newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/index.js

```diff
@@ -175,6 +175,7 @@ export default class LinkEvent extends React.Component<EventRendererProps, *> {
               <ExternalEventsAutoComplete
                 project={this.props.project}
                 value={target}
+                sceneName={this.props.scope.layout.getName()}
```

Comment:
The layout is not guaranteed to be present (for example in events based functions)

```suggestion
                sceneName={this.props.scope.layout ? this.props.scope.layout.getName() : ""}
```

Reply:
Remember to add the prop to the flow type as well:

https://github.com/4ian/GDevelop/pull/3228/files#diff-5213df436229c337cb945f865908464ae1d144072d017e97da8f042a327bfc12R31-R38